### DISCUSSION TITLE
Readings in prep for week 9

### DISCUSSION THREAD

**Discussion by benmiller314** [benmiller314/dsam2023fall#11] [Reading responses etc]
After reading [Arnold and Tilton](https://dhdebates.gc.cuny.edu/read/untitled-f2acf72c-a469-49d8-be35-67f9ac1e3a60/section/a2a6a192-f04a-4082-afaa-97c76a75b21c#ch24) on the role of statistics in DH, Alberto Cairo on [visualization](https://learning.oreilly.com/library/view/the-truthful-art/9780133440492/ch05.html) and [exploratory data analysis](https://learning.oreilly.com/library/view/the-truthful-art/9780133440492/ch06.html), and D'Ignazio and Klein [on the writer's role in presenting and interpreting ("speaking for") data](https://ebookcentral.proquest.com/lib/pitt-ebooks/reader.action?docID=6120950&ppg=162), please write a short response to let us know where your mind went. For example, you might...

* Highlight new ideas or key terms, especially those that you expect will be useful in thinking about digital projects.
* Highlight sentences, passages, or methods that struck you as especially well done, perhaps worth imitating.
* Link outward to something a passage reminds you of, explaining where you're leading us.
* Pose questions to your classmates about a passage, especially (but not exclusively) if something is confusing.
* Respond to any of the above – which you can now do directly, using threads.

Where possible, please try to point us to at least one particular passage from the text you're responding to – quoting if you can, giving a timestamp or page reference either way – as a way to help keep our discussion grounded.

**Comment by chloe-cmu** [comment]
My personal rule of thumb for evaluating infographics has always been evaluating how long it takes for me to understand what the graphic is trying to tell me. Writing that out feels a little embarrassingly self-centered – though this was more of an internal rule rather than anything I actively critiqued anyone else on. Visual representations of data should be bold, crisp, and easy to understand. Today’s readings were a good reminder that it’s often not so simple. My own assessment of infographics falls in line with Cairo’s hierarchy of elementary perceptual tasks (or, more accurately, that concept helped me understand a lot of the criteria I have based previous judgements on unconsciously). But as D’Ignazio and Klein’s reminded me, context is always a central consideration. Many infographics may make more intuitive sense to me because I share the assumptions of the person who built them. It’s important to think carefully about where data comes from and how it’s represented. Arnold and Tilton also remind me of the importance of knowing my own limits as a thinker. As a young scholar in a precarious job market, my instinct is always to try to learn more, do more, understand more. But there’s a limit to how much any one person can understand a huge variety of disciplines, especially in the digital humanities. This reminder highlights the continuing importance of collaboration in these kinds of projects. 

I found the data visualization catalogue that Cairo linked to helpful – I had never seen a good number of those graphs before, like the Nightingale Rose Chart, spiral plot, or sunburst diagram. I’m not sure if I would use any of the more complex options for this semester’s project, but it was still very helpful to see and think about.  I also found D’Ignazio and Klein’s suggestion of a data user guide helpful – just to have a place, beyond the minutia of a data dictionary, to provide a narrative context for a particular project.

**Comment by amj169** [comment]
These chapters came at a good time because they helped me think about the ways in which I’ll continue a similar conversation with my students about choosing the mediums that are right for their projects and how they’ll choose to present those projects. They’re in the initial research stages and I’ve begun to talk to them about different types of multimodal work and how we can approach that. I find Cairos’s offerings about how we should be carefully selecting data visualizations and D’Ignazio and Klien’s conversation about context (especially in framing Borgman’s “knowledge infrastructure” lens of the world) to be a perfect union in order to help discuss choosing vehicles to represent the multimodal work I’m asking them to take part in. I find these resources especially useful in talking about why isolation as a research practice can pose inequitable and even potentially harmful obstacles, unintentionally. I’m trying to get them to consider how we can responsibly think about what our end products might look like by asking questions about ownership, authorship, history, and audience, first.  

For example, some students are interested in creating mapping data visualizations but aren’t sure what they’re going to map and haven’t developed their research questions yet. Right now they’re doing source work to acquaint themselves to the happenings in the fields they’re beginning to situate themselves in, so these articles really came at a great time for me because I can use them to help navigate the conversations about why numbers can’t actually speak for themselves. I think I’ll give them some examples from CH 5 to illustrate how your research can inform your public facing product in ways that are more logical (and responsible) than others. 

I’m also interested in talking about consent from online users (say you’re collecting Reddit or Instagram comments, which some of my students are interested in doing). I still feel weary in some ways about scraping comment sections to use for their work even though the posters published them on public forums. Is this enough consent? This I think is very subjective and the context has to be considered, but I still wonder to what extent I can help translate their desires to work with certain online communities to the feminist methodological framework that D’Ignazio and Klien put forth.

**Comment by Khushboobhutani** [comment]
This week’s readings speak to the conundrum that I have been facing with respect to the representation of data. The well-structured dialogues between statistics and humanities laid out a huge range of possibilities to choose from. Cairo’s piece, however, forced me to rethink my stance and examine the context and idiosyncrasies of my data set. Given the nature of my project, it is difficult to reduce the afflicted population into numbers and most Palestinians have repeatedly expressed their disdain towards such an approach. While I second Arnold and Tilton’s claim that using a “general purpose statistical programming language increases the available set of methodological approaches to studying humanities data,” I also acknowledge the reluctance of my subjects for whom the idea of being translated into numbers or statistics runs contrary to apt representation. This leads me to ponder upon what other universally accessible and acceptable approaches would befit such situations. If one needs to find right visualizations, they need to start asking right questions.

**Comment by SidraNArshad** [comment]
There is an anecdote included in the beginning of Cairo’s “Basic Principles of Visualization” that discusses a University of Chicago professor who changed the point value of an exam, which served to lower his class’s average score on the exam. Despite this negative effect on their grades, students were still happier with the supposed change that the professor made because “96 over 137 still feels higher than 72 over 100.” I think the use of the word “feel” here is significant in its larger connection to a trend encapsulated in a popular TikTok meme that initially came to be known as “girl math.” (https://www.cnn.com/2023/10/07/us/girl-math-boy-social-media-cec/index.html) The link I’ve included outlines the basics of “girl math” which comes down to spending more money or time on a product or act than necessary, but doing it in such a way that feels like you aren’t. The meme has spread to incorporate other groups—I’ve seen “Muslim math”, “parent math”, and “South Asian math” come across my for you page. Unfortunately, something that the linked article also highlights is the problematic commandeering of the trend to perpetuate problematic gender stereotypes and misogynistic talking points. The basic joke of “girl math” has also been used to demean women as well—associating the joke with our apparent financial incompetence. This might be a stretch, but I feel like girl math’s rise and fall on the internet’s established timeline of innocent joke distorted and hijacked to be used for bigotry and violence connects to our larger discussions of misrepresentation and context in a way that I’m struggling to articulate.

**Comment by MustafaKandil** [comment]
The theme this week focuses on the critical aspects of data analysis and visualization. In one of the readings, Exploratory Data Analysis (EDA) is emphasized as a fundamental approach to comprehend and reveal patterns within data sets. Additionally, the significance of statistics, specifically exploratory data analysis, data visualization, and statistical computing, is underscored for a comprehensive understanding of humanities data, leading to more insightful research outcomes.

Furthermore, another reading sheds light on the fundamental principles of visualization. It raises an important question: How can one ensure the appropriate selection of a graphic form to effectively represent data? This decision is crucial and demands careful consideration. Personally, I concur with the notion that visually encoding data is pivotal. Our learning process as humans is intricately intertwined with our sensory experiences. Consequently, our senses significantly influence how we perceive, interpret, and comprehend the world. These insights highlight the essential role of integrating digital methods in humanities research to foster thoughtfulness and effective learning.

**Comment by Rezaetmd** [comment]
I also found Cairo's paper on visualization interesting as it delves into the significance of selecting appropriate graphic formats for data visualization, structuring data displays, and efficiently conveying information. It underscores the importance of factoring in the specific task or message, accurately representing data, selecting meaningful references and scales, and experimenting with various visual formats. Furthermore, the paper addresses the constraints of particular chart types and examines how visual presentations can influence decision-making. Though my project would not entail such visualization, I still think that if one needs to go through multiple sources of data and then visualize it for (general) audience, this chapter would definitely have some good points to make.
